templates/crane.conf.erb

```diff
@@ -0,0 +1,9 @@
+# =========================
+# Pulp Crane Configuration
+# =========================
+
+[general]
+debug: false
+#data_dir_polling_interval: 60
```

Comment:
This is commented out and cannot be modified by a user, mind either removing it entirely or making it configurable?